tests/conftest.py

```diff
@@ -116,3 +133,109 @@ def structurally_valid_vote():
             'timestamp': '1111111111'
         }
     }
+
+
+@pytest.fixture
+def setup_database(request, restore_config, node_config):
```

Comment:
I'll update `setup_database` and `cleanup_tables` to match what was done in #953.